lib/game.rb

```diff
@@ -0,0 +1,89 @@
+class Game
+  include Constable
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    return player_1 if board.turn_count.even?
```

Comment:
You can use a ternary to implicitly return a value `@board.turn_count.even? ? @player_1 : @player_2`.